server/routes/checkAnswers.js

```diff
@@ -3,7 +3,8 @@ const bodyParser = require('body-parser')
 const flash = require('connect-flash')
 const asyncMiddleware = require('../middleware/asyncMiddleware')
 
-module.exports = function Index({ formService, authenticationMiddleware }) {
+// eslint-disable-next-line
```

Comment:
the eslint-disable warnings should only be used very sparingly - can you fix the issue which has lead you to put this in?

Reply:
it was a line I had forgotten to take out. I had initially included the lint disable  because I was erroneously 'requiring' offenderService in line 5 as well as passing it in as an arg in line 7. 
However have removed the 'require' so ok to take out the lint disable